lib/katello/tasks/upgrades/3.14/republish_deb_metadata.rake

```diff
@@ -0,0 +1,24 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.14' do
+      desc "Regenerate the metadata for repositories affected by pulp_deb migration 0004"
+      task :republish_deb_metadata => ["environment"] do
+        User.current = User.anonymous_api_admin
+        input_file_name = "/var/lib/pulp/0004_deb_repo_republish_candidates.txt"
+        if File.file?(input_file_name)
+          pulp_ids = []
+          File.open(input_file_name, "r") do |file_handle|
+            file_handle.each_line do |pulp_id|
```

Comment:
Isn't this equivalent?
```ruby
pulp_ids = file_handle.each_line.map(&:strip)
```

Reply:
Or even shorter :smile::
```ruby
pulp_ids = File.read(input_file_name).each_line.map(&:strip)
```

Reply:
I tried to use @m-bucher's version, but I have yet to test it.

Reply:
I tested and it worked.